go/internal/services/keys/validation.go

```
@@ -29,6 +29,11 @@ func (k *KeyVerifier) withCredits(ctx context.Context, cost int32) error {
 		return nil
 	}
 
+	// Key has unlimited requests if set to NULL
+	if k.Key.RemainingRequests.Valid == false {
```

golangci-lint (2.2.2)

S1002: should omit comparison to bool constant, can be simplified to !k.Key.RemainingRequests.Valid

(staticcheck)